segment_osicon_test.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestOsIcon(t *testing.T) {
+	env := new(MockedEnvironment)
+	env.On("getRuntimeGOOS", nil).Return("windows")
+	osicon := &osicon{env: env}
```

Comment:
You can also assign properties to validate the exact icon, so differ between set and not set.
See [here](https://github.com/JanDeDobbeleer/oh-my-posh3/blob/main/segment_exit_test.go#L30) for example.

Reply:
wil do, thanks.